app/src/main/java/org/mozilla/focus/fragment/BrowserFragment.java

```diff
@@ -59,7 +60,7 @@
     private static final int ANIMATION_DURATION = 300;
 
     private static final String ARGUMENT_SESSION_UUID = "sessionUUID";
-    private static final int SCROLL_VELOCITY = 1200;
+    private static final int SCROLL_VELOCITY = 45;
```

Comment:
nit: this isn't really a `VELOCITY` – it's a "velocity -> scroll velocity" multipler.